version/1/4/heb/code_of_conduct.md

```diff
@@ -0,0 +1,65 @@
+# קוד התנהגות לתורם
+
+## ההתחייבות שלנו
+
+לטובת עידוד של סביבה פתוחה ומקבלת, אנחנו בתור התורמים והמתחזקים מתחייבים להפוך 
+את ההשתתפות בפרויקט שלנו ואת הקהילה שלנו לחוויה נטולת הטרדות לכל אדם באשר הוא
+ללא הבדלי גיל, משקל, מגבלות פיזיות, מוצא אתני, הזדהות מגדרית, רמת ניסיון, לאום, 
+מראה חיצוני, גזע, דת, זהות ונטיה מינית.
+
+## הסטנדרטים שלנו
+
+דוגמאות של התנהגות התורמת לסביבה חיובית כוללות:
+
+* שימוש בשפה מקבלת ומברכת
+* כיבוד של נקודות מבט וחוויות שונות
+* קבלה של ביקורת בהבנה
+* התמקדות במה שהכי טוב לקהילה
+* הפגנת אמפתיה כלפי חברים אחרים מהקהילה
+
+דוגמאות של התנהגות לא מקובלת כוללות:
+
+* שימוש בשפה מינית, תמונות גסות, או הטרדה מינית
+* הטרלה, תגובות מעליבות/משפילות, תקיפה אישית או פוליטית
+* הטרדה פומבית או אישית
+* פרסום מידע אישי של אחרים, כגון כתובת פיזית או אלקטרונית, בלי אישור מפורש
+* התנהגות כלשהי שאינה סבירה בסביבת עבודה מקצועית
+
+## האחריויות שלנו
+
+מתחזקי הפרויקט אחראיים להבהיר את הסטנדרטים להתנהגות מקובלת ומצופים לפעול באופן 
+מתאים והוגן בתגובה לכל אירוע של התנהגות בלתי מקובלת.
+
+יש למתחזקי הפרויקט את הזכות והאחריות להסיר, לערך, או לדחות תגובות, קומיטים, קוד, 
+עדכוני וויקי, נושאים, או תרומות אחרות שאינם מיושרות עם קוד ההתנהגות הנ״ל, או לחסום
+באופן זמני או קבוע כל תורם להתנהגויות אחרות שהם מוצאים אי הולמות, מאיימות, פוגעניות,
+או פוגעות.
+
+
+## היקף
+
+קוד ההתנהגות הנ״ל תקף גם במרחבי הפרויקט וגם במרחבים ציבוריים כאשר אינדיבידואל מייצג
+את הפרויקט או את הקהילה שלו. דוגמאות לייצוג פרויקט או קהילה כוללות שימוש בכתובת 
+אימייל רשמית של הפרויקט, פרסום דרך חשבון מדיה חברתית שלו, או התייצבות כנציג ממונה
+במפגש באינטרנט או באופן לא מקוון. ייתכן שהייצוג של הפרויקט יורחב ויובהר על ידי מתחזקיו.
+
+## אכיפה
+
+
+תופעות של התנהגות מעליבה, מטרידה, או לא מקובלת יכולות להיות מדווחות על ידי יצירת קשר
+עם הקבוצה של הפרויקט ב [הכנס כתבות אימייל]. כלל התלונות יבחנו ויחקרו ויענו על ידי 
+תגובה שצוות הפרויקט רואה כמתאימה והכרחית לנסיבות. צוות הפרויקט מחויב לשמור על חשאיות
+בנוגע למדווח של התקרית. פרטים נוספים של מדינויות אכיפה ספציפיות ידווחו בנפרד.
+
+מתחזקי פרויקט שלא עוקבים ואוכפים את קוד ההתנהגות ברצון טוב עתידים להשלכות זמניות או
+קבועות כפי שיקבעו על ידי הנהגת פרויקט זה.
```

Comment:
`כפי שיקבעו על ידי הנהגת פרויקט זה` says "as determined by other members of this project's"

More correct:

`כפי שיקבעו על ידי הנהגת הפרויקט`, which reads more like "as determined by other members of the project's leadership" as originally depicted.

Actually none of the above strictly says "by other members of the project's leadership" but instead "by the project's leadership".

An even more accurate translation would be:

`כפי שיקבעו על ידי חברי הנהגת הפרויקט האחרים`